### PR TITLE
5152 / Openfin Notification indicator colors

### PR DESCRIPTION
**Pull request by algreasley**
rtc-5152-notifications-fx-accepted-rejected-for-PR
rtc-5152-notifications-credit-quote-accepted-for-PR

### CODE REVIEW

**Comment by SHolleworth** [comment]
We could use the styleguide colours for buy and sell for pill background colours `colors.spectrum.uniqueCollections.Buy.base`

**Review by snony14 — DISMISSED** [review]

File: src/client/src/notifications.web.ts [hunk 1]
```diff
@@ -1,107 +1,88 @@
...
+    body: `${trade.direction} ${tradeCurrencyDetails}`,
+    icon: fxIconUrl,
     dir: "ltr",
-    data: notification,
-    // tag: "trade", TODO: investigate why this field causes malfunctions on certain versions of chrome
   }
 
-  new Notification(title, options)
+  const fxNote = new Notification(title, options)
+  fxNote.onclick = (notification) => console.log(notification)
```
Comment (SHolleworth) [inline src/client/src/notifications.web.ts:33]: Did you leave this intentionally?

**Review by SHolleworth — DISMISSED** [review]

File: src/client/src/notifications.web.ts (hunk 1 above)
Comment (algreasley) [reply src/client/src/notifications.web.ts:33]: nope 🤦🏻  ... a test for onClick, works on desktop but not iOS unfortunately -- will remove

Comment (algreasley) [reply src/client/src/notifications.web.ts:33]: might use this in future for OpenFin-like behaviour - obv atm this is just simple blotter highlighting

**Review by SHolleworth — APPROVED** [review]

**Pull request merged**